recommender/errors.py

```diff
@@ -11,7 +11,12 @@ def message(self):  # pragma: no cover
         return "Invalid recommendation panel id error"
 
 
-class UntrainedPreAgentError(Exception):
+class NoActorModelForK(RecommendationEngineError):
+    def message(self):  # pragma: no cover
+        return "There is no Actor model for used K (number of services in recommendation)."
```

Comment:
"for used K" -> "for K used"

Reply:
"used" word is related to the "K" - not to the "Actor Model" so I think the grammar of this sentence is correct.